pwndbg/disasm/arch.py

```diff
@@ -1,11 +1,12 @@
+import capstone as C
 import gdb
-from capstone import *  # noqa: F403
 
 import pwndbg.lib.memoize
 import pwndbg.symbol
 
 debug = False
 
+# TODO: Do these need to change to 'C.CS_'?
 groups = {v: k for k, v in globals().items() if k.startswith("CS_GRP_")}
 ops = {v: k for k, v in globals().items() if k.startswith("CS_OP_")}
 access = {v: k for k, v in globals().items() if k.startswith("CS_AC_")}
```

Comment:
With this PR, these arrays are now empty, but no tests are failing. That means we need to improve our test coverage first before merging this PR.

Reply:
@disconnect3d would you be able to help with testing this?

Reply:
Yeah, I can look at it probably during the weekend after ASIS CTF